XenAdmin/SettingsPanels/NRPEEditPage.cs

```diff
@@ -29,26 +29,27 @@
  */
 
 using System;
+using System.Collections.Generic;
 using System.Drawing;
+using System.Text.RegularExpressions;
 using System.Windows.Forms;
-using XenAPI;
 using XenAdmin.Actions;
-using System.Collections.Generic;
 using XenAdmin.Core;
-using System.Text.RegularExpressions;
+using XenAdmin.Actions.NRPE;
+using XenAPI;
+
 
 namespace XenAdmin.SettingsPanels
 {
     public partial class NRPEEditPage : UserControl, IEditPage
     {
         private static readonly log4net.ILog log = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
 
-
         private static readonly Regex REGEX_IPV4 = new Regex("^((25[0-5]|2[0-4]\\d|[01]?\\d\\d?)\\.){3}(25[0-5]|2[0-4]\\d|[01]?\\d\\d?)");
         private static readonly Regex REGEX_IPV4_CIDR = new Regex("^([0-9]{1,3}\\.){3}[0-9]{1,3}(\\/([0-9]|[1-2][0-9]|3[0-2]))?$");
         private static readonly Regex REGEX_DOMAIN = new Regex("^(((?!-))(xn--|_)?[a-z0-9-]{0,61}[a-z0-9]{1,1}\\.)*(xn--)?([a-z0-9][a-z0-9\\-]{0,60}|[a-z0-9-]{1,30}\\.[a-z]{2,})$");
 
-        private readonly bool IsHost = true;
+        private bool _isHost;

```

Comment:
Naming convention: for private instance class fields we use camelcase with undescore. I only changed this instance as an example, but there are several more fields in these files that could benefit from the change.